File: w09/d03/quin_cogdell/mail_objects_app/spec/javascripts/postcard_spec.js
describe("a Postcard", function(){

  beforeEach(function(){
    postcard = new Postcard();
    message = "Hi Mom";
    postcard.message(message);
  });

  it("has a messages that is a string", function(){
    expect(postcard.message).toBe("Hi Mom");
    expect(typeof(postcard.message)).toBe("string");
  });



});

File: w09/d01/amy_ruan/mailer/spec/postcard_spec.rb
require_relative('../lib/postcard')

describe Postcard do
  it "has a ship_to_address" do
    postcard = Postcard.new(ship_to_address: "902 Broadway")
    expect(postcard.ship_to_address).to eq "902 Broadway"
  end

  it "should have a message" do
    postcard = Postcard.new(message: "whats up")
    expect(postcard.message).to eq("whats up")
  end
end